openvoxserver-ca.gemspec

```
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "facter", [">= 2.0.1", "< 5"]
+  spec.add_runtime_dependency "openfact", [">= 2.0.1", "< 5"]
```

At least for now we need:

```suggestion
  spec.add_runtime_dependency "openfact", [">= 5.0.0.rc0", "< 6"]
```